test/finset_repr.lean

```diff
@@ -1,11 +1,13 @@
 import Mathlib.Data.Finset.Sort
 
-run_cmd Lean.Elab.Command.liftTermElabM do
-  unsafe guard <|
-   (repr (0 : Multiset (List ℕ)) |>.pretty 15) = "0"
-  unsafe guard <|
-    (repr ({[1, 2, 3], [4, 5, 6]} : Multiset (List ℕ)) |>.pretty 15) = "{[1, 2, 3],\n [4, 5, 6]}"
-  unsafe guard <|
-    (repr (∅ : Finset (List ℕ)) |>.pretty 15) = "∅"
-  unsafe guard <|
-    (repr ({[1, 2, 3], [4, 5, 6]} : Finset (List ℕ)) |>.pretty 15) = "{[1, 2, 3],\n [4, 5, 6]}"
+-- FIXME temporarily commented out: unsafe can't run outside of a declaration.
```

Comment:
Is this a bug, or a deliberate change? Can you reference the PR that introduced the change in this FIXME?

Reply:
Scott said this a bit further up:

> These will be restored in nightly-2024-02-15. The fix is already in.

So they should be uncommented soon

Reply:
Let's put the lean 4 pr number in the comment then

Reply:
It's resolved already, I'll restore these tests when nightly-2024-02-15 lands.

Reply:
Added links.

Reply:
Thanks!